psamm/datasource/sbml.py

```diff
@@ -713,20 +732,40 @@ def write_model(self, file, model, reactions, compounds):
 
         # Create list of reactions
         reactions = ET.SubElement(model_tag, self._sbml_tag('listOfReactions'))
-        for reaction in model.reactions:
+        # Create mapping for reactions containing flux limit definitions
+        for flux_limit in model.parse_limits():
+            eq_id = flux_limit[0]
```

Comment:
I would change this from `eq_id` to `reaction_id`. It's more consistent with the naming in other parts of PSAMM. We don't use the term _equation ID_ elsewhere but we use _reaction ID_.